doc/geany.txt

```diff
@@ -2542,6 +2542,8 @@ msgwin_messages_visible           Whether to show the Messages tab in the      t
                                   Messages Window
 msgwin_scribble_visible           Whether to show the Scribble tab in the      true        immediately
                                   Messages Window
+warn_on_project_close             Whether to show a warning when opening       true        immediately
```

Comment:
The option name and description doesn't match the rest of the feature. I guess this is a copy&paste mistake from #2949.

Reply:
Possibly copy/paste.  Possibly I got confused by too many git branches.  I've rebased and fixed this (maybe, if I didn't get confused again).